odoo/addons/base/tests/common.py

```diff
@@ -32,6 +32,8 @@ def setUpClass(cls):
         if independent_user:
             cls.env = cls.env(user=independent_user)
             cls.user = cls.env.user
+        else:
+            cls.env.user.group_ids += cls.get_default_groups()
```

Comment:
@Gorash Didn't you make something else to handle this?

I think this should not be added to tests, but each test should declare what groups its default user has giving it minimal rights for what is being tested.